webpack/environments/baseConfig.js

```diff
@@ -0,0 +1,117 @@
+// @ts-check
+const path = require("path")
+const webpack = require("webpack")
+const {
+  NODE_ENV,
+  basePath,
+  isCI,
+  isDeploy,
+} = require("../../src/lib/environment")
+const { getEntrypoints } = require("../utils/getEntrypoints")
+
+const baseConfig = {
+  devtool: "cheap-module-eval-source-map",
+  entry: {
+    webpack: [
```

Comment:
We should actually probably move this to the devConfig. I don't think you want to generate this for the production build.

Reply:
If you _do_ move this, it'd be good to run the build in both modes just to be sure the bundles are output like you expect.